README.md

```diff
@@ -103,7 +103,13 @@ Pretty easy. Just check out the code, then run:
 $ npm install
 ```
 
-That'll pull down the dependencies. Then run:
+That'll pull down the dependencies. 
+
+```bash
+$ bower install
```

Comment:
Surprised this was necessary, actually—`bower install` should be a postinstall hook…